_rules/error-field-is-not-identified-using-color-only

```diff
@@ -0,0 +1,107 @@
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [F81 - Failure of Success Criterion 1.4.1 due to identifying required or error fields using color differences only](https://www.w3.org/WAI/WCAG21/Techniques/failures/F81.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The presence of errors is conveyed through text error message.
+
+```html
+<label for="first_name">First Name</label>
+<input type="text" id="first_name" autocomplete="given-name" style="border:1px solid #dd0000" aria-invalid="true" aria-describedby="first_name_error_msg">
+<p id="first_name_error_msg" class="error_msg" style="color:#dd0000">This field is empty. Enter your first name.</p>
+```
+
+#### Passed Example 2
+
+The presence of errors is conveyed also through other CSS properties:
+- font-weight: bold AND;
+- border-width: 3px.
+
+```html
+<label for="first_name" style="color:#dd0000">First Name</label>
+<input type="text" id="first_name" autocomplete="given-name" style="border:3px solid #dd0000" aria-invalid="true">
+<label for="middle_name" style="color:#333333">Middle Name (optional)</label>
+<input type="text" id="middle_name" autocomplete="additional-name" style="border:1px solid #b0b0b0" value="Michael">
+<label for="last_name" style="color:#333333">Last Name</label>
+<input type="text" id="last_name" autocomplete="family-name" style="border:1px solid #b0b0b0" value="Doe">
+```
+
+#### Passed Example 3
+
+The presence of errors is conveyed through:
+- lightness (difference in relative luminance between the colors leads to a contrast ratio greater than 3:1) AND;
+- font-weight: bold.
+
+```html
+<label for="first_name" style="color:#dd0000; font-weight:bold;">First Name</label>
+<input type="text" id="first_name" autocomplete="given-name" style="border:1px solid #dd0000" aria-invalid="true">
+<label for="last_name" style="color:#000">Last Name</label>
+<input type="text" id="last_name" autocomplete="family-name" style="border:1px solid #000" value="Doe">
+```
+
+### Failed
+
+#### Failed Example 1
+
+The presence of errors is conveyed through color (hue) differences only.
+
+```html
+<label for="first_name" style="color:#dd0000">First Name</label>
+<input type="text" id="first_name" autocomplete="given-name" style="border:1px solid #dd0000" aria-invalid="true">
+<label for="last_name" style="color:#333333">Last Name</label>
+<input type="text" id="last_name" autocomplete="family-name" style="border:1px solid #b0b0b0" value="Doe">
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+There are no error fields.
+
+```html
+<p>This is a paragraph.</p>
+```
```

Comment:
You need to add the reference list of the terms used in the rule.
For example, you use `[visible][]`, so there should be here a definition:
```
[visible]: #visible 'Definition of Visible'
```
(see how it's done in other rules files